tests/llm/test_prepared_request_contract.py

```
@@ -252,3 +253,83 @@ async def test_api_client_delegates_prepared_request_to_handler(
+        client_preference="openrouter",
+    )
+    client.model_config.max_context_window_tokens = 272000
+    client.model_config.max_output_tokens = None
+    client.system_prompt = None
+    client.client_handler = handler
+    client.logger = logging.getLogger(__name__)
+    client._last_error = None
+    client._last_response_result = None
+    client._clear_handler_error = lambda: None
+    client._get_handler_error = lambda: None
+    client._build_response_result = lambda **kwargs: None
```

Ruff (0.15.13)

[warning] Unused lambda argument: `kwargs`

(ARG005)